Tests/CustomEntitySearchTest/TestData.cs

```diff
@@ -267,5 +269,183 @@ public static async Task<WebApiSkillResponse> GeneratePayloadRequest(String inpu
                 request, new LoggerFactory().CreateLogger("local"), new Microsoft.Azure.WebJobs.ExecutionContext()));
             return (WebApiSkillResponse)response.Value;
         }
+
+        public static void supportedTextandWordsTempInitializer()
+        {
+            supportedTextandWords.Add("Greek", new string[]
+                {
+                    @"""Tου Αντώνη Ρέλλα - Θα πρέπει να γίνει κατανοητό ότι οι αποκλεισμοί των αναπήρων εκκινούν, έτσι κι αλλιώς, από τις θεσμοθετημένες πρακτικές του κράτους και τα εμπόδια στο δομημένο περιβάλλον. Πώς, λοιπόν, η κυβέρνηση θα κάνει πράξη την ανεξάρτητη διαβίωση""",
```

Comment:
Why do these strings have quotes inside them?

Reply:
Prior to the .NET version I was using serialized strings. There should not be any quotes aside from the beginning and ending quotes.